src/VfacTmdb/Items/TVNetwork.php

```diff
@@ -28,8 +28,6 @@
  */
 class TVNetwork extends Item implements TVNetworkInterface
 {
-    use ElementTrait;
```

Comment:
I took this out as TVNetworks don't have posters or backdrops so it seemed unneeded.